solc/CommandLineInterface.cpp

```diff
@@ -416,8 +418,10 @@ void CommandLineInterface::readInputFilesAndConfigureRemappings()
 				auto infile = boost::filesystem::path(path);
 				if (!boost::filesystem::exists(infile))
 				{
-					cerr << "Skipping non-existent input file \"" << infile << "\"" << endl;
-					continue;
+					if(! m_args.count(g_argIgnoreMissingFiles)){
+						cerr << "Non-existent input file " << infile << endl;
+						exit(1);
```

Comment:
Do other parts of the code directly call exit? If not, rather return a bool.

Reply:
@chriseth  I couldn't find other instances so I can return a bool.  It's gonna cause a little more work further up the stack which is not a big deal, simply expands the scope for number of functions this change touches.  I'll have it return a bool.